packages/components/inputs/rich-text-utils/src/localized/localized.ts

```diff
@@ -0,0 +1,64 @@
+import { warning } from '@commercetools-uikit/utils';
+import uniq from 'lodash/uniq';
+import html from '../html';
+import isRichTextEmpty from '../is-empty';
+
+type TLocalizedString = {
```

Comment:
I guess this type would shape objects like this:
```
{
  messages: 'en'
}
```

But I think that, in the context it is used, the expected object is like this:
```
{
  'en': 'Hello',
  'es': 'Hola'
}
```
so the type should be:
```
type LocalizedString = {
  [language: string]: string;
}

// OR
type TLocalizedString = Record<string, string>;
```

What do you think?

Reply:
This makes sense. Thanks for pointing it out.